### DISCUSSION TITLE
request not response

### DISCUSSION THREAD

**Discussion by nilusr** [qruto/laravel-wave#31] [Q&A]
I downloaded the demo code and deploy it to the local environment. For the first time, I can open the page normally and sSE has also established channels. If refresh the page at this time or opens a new tab, the status is always pending and doesn't timeout yet.

<img width="1651" alt="image" src="https://github.com/qruto/laravel-wave/assets/51184850/6827d57b-92cd-4ea3-95a1-73581888e6fd">

**Comment by slavarazum** [comment]
You can find a live demo on https://wave.qruto.dev

If you want to run your own server, don't forget to set your Twitter API keys.

**Reply by nilusr** [reply]
actually I'm not using twitter
<img width="636" alt="image" src="https://github.com/qruto/laravel-wave/assets/51184850/728690de-4a11-4775-be9b-f910d8f67f19">

**Reply by slavarazum** [reply]
Which local environment do you use?
Try to update to the latest package version.

**Comment by slavarazum** [comment]
If you still have issues with the demo application, feel free to reopen the discussion and provide more details about your case.
